scraping/scraping/spiders/_base.py

```diff
@@ -131,14 +129,21 @@ def parse_PRODUCT(self, response: Union[SplashJsonResponse, ScrapyHttpResponse])
         Args:
             response (SplashJsonResponse): Response from a performed request
         """
+
+        request_meta_information = response.meta.get("amazon_price", None)
+        if request_meta_information is not None:
+            meta_information = self.meta_data | {"amazon_price": request_meta_information}
```

Comment:
... here we then need to: `self.meta_data | request_meta_information` because it is an dictionary already

Reply:
Changed in [74f677d](https://github.com/calgo-lab/green-db/pull/65/commits/74f677d372ac18391ade43d100136d7839f61e54)